src/common/src/result_format/c_checker.cpp

```diff
@@ -257,41 +254,41 @@ void cChecker::ClearParams()
 }
 
 /*
```

Comment:
```suggestion
/**
```
To enable automatic docs, I suspect this is required. To be repeated in the rest of the file.